Telereso/Classes/Telereso.swift

```diff
@@ -165,6 +175,31 @@ public struct Telereso{
         return local
     }
     
+    static private func getDrawableValue(_ drawableKey: String) -> String {
```

Comment:
please rename to `getRemoteDrawables` , it's deferent than `getRemoteDrawable` so should be fine